homework4/task1-2.js

```diff
@@ -2,23 +2,22 @@
 
 //---------------task 1------------------
 
+function addThemAll (...allArg ){
+    return allArg.reduce((a, b)=>a+b);
```

Comment:
I have tried to add default values like that --  allArg.reduce((a = 1, b = 1)=>a+b); , but error occurs "Reduce of empty array with no initial value"